genestack_client/genome_query.py

```diff
@@ -1,3 +1,12 @@
+from __future__ import absolute_import
+from __future__ import division
+from __future__ import print_function
+from __future__ import unicode_literals
+from future import standard_library
+standard_library.install_aliases()
+from builtins import str
+from builtins import *
+from builtins import object
```

Comment:
Looks like `import *` is enough
```suggestion
from builtins import *
```

There are many other files with the same issue.